gen/safeq

```diff
@@ -141,25 +136,32 @@ perunServicesInit::finalize;
 ##############################################################################
 
 sub processUsers {
-	my $memberAttributes = shift;
-	my $resourceAttributes = shift;
-	my $login = $memberAttributes->{$A_USER_LOGIN};
-
-	unless(defined $usersByLogin->{$memberAttributes->{$A_USER_LOGIN}}) {
-	my %tmp =  ( %$memberAttributes, %$resourceAttributes );
-		$usersByLogin->{$memberAttributes->{$A_USER_LOGIN}} = $memberAttributes;
-		$usersByLogin->{$memberAttributes->{$A_USER_LOGIN}}->{$A_R_PREPAID} = $resourceAttributes->{$A_R_PREPAID};
-		$usersByLogin->{$memberAttributes->{$A_USER_LOGIN}}->{$A_R_PRINT_FOR_FREE} = ( $resourceAttributes->{$A_R_PRINT_FOR_FREE} || 0) ;
+	my $memberId = shift;
```

Comment:
See where you print `role:` in LDIF file. You can probably use `sort keys ...`  to get comparable outputs. It's same for `billingcode:` or `card:` fields.

Reply:
I'm waiting for confirmation from service owner that we can sort those attributes, so please wait with that cahnge. Also it seems, that `safeq5` version can be deleted instead - also waiting for confirmation, so we won't merge this yet.